akka-distributed-data/src/test/scala/akka/cluster/ddata/LWWRegisterSpec.scala

```diff
@@ -13,8 +13,8 @@ import org.scalatest.WordSpec
 class LWWRegisterSpec extends WordSpec with Matchers {
   import LWWRegister.defaultClock
 
-  val node1 = UniqueAddress(Address("akka.tcp", "Sys", "localhost", 2551), 1)
-  val node2 = UniqueAddress(node1.address.copy(port = Some(2552)), 2)
+  val node1 = UniqueAddress(Address("akka.tcp", "Sys", "localhost", 2551), 1L)
+  val node2 = UniqueAddress(node1.address.copy(port = Some(2552)), 2L)
```

Comment:
that's just annoying if one must write it like that

Reply:
Yes, removed the '-Ywarn-numeric-widening'.

These specific ones are not regular 'numeric widening' warnings though, but calls to our deprecated API:

```
  // for binary compatibility
  @deprecated("Use Long UID apply instead", since = "2.4.11")
  def apply(address: Address, uid: Int) = new UniqueAddress(address, uid.toLong)
```

Once the deprecated methods are removed we can use regular integers again.

Reply:
ah, I see